cmd/kubeadm/app/phases/upgrade/preflight.go

```diff
@@ -80,20 +80,19 @@ func checkUnsupportedPlugins(client clientset.Interface) error {
 	if err != nil {
 		return err
 	}
-	unsupportedCoreDNS, err := migration.Unsupported(currentInstalledCoreDNSversion, kubeadmconstants.CoreDNSVersion, corefile)
+	unsupportedCoreDNS, err := migration.Unsupported(currentInstalledCoreDNSversion, currentInstalledCoreDNSversion, corefile)
 	if err != nil {
 		return err
 	}
 	if unsupportedCoreDNS != nil {
-		var UnsupportedPlugins, UnsupportedVersion string
+		var UnsupportedPlugins []string
 		for _, unsup := range unsupportedCoreDNS {
-			UnsupportedPlugins = unsup.Plugin
-			UnsupportedVersion = unsup.Version
-
-		}
-		if UnsupportedPlugins != "" || UnsupportedVersion != "" {
-			return errors.New("there are unsupported plugins in the CoreDNS Corefile")
+			UnsupportedPlugins = append(UnsupportedPlugins, unsup.Plugin)
 		}
+		klog.V(1).Infoln("The corefile contains plugins that kubeadm/CoreDNS does not know how to migrate.")
+		klog.V(1).Infoln("Each plugin listed should be manually verified for compatibility with the newer version of CoreDNS. Once ready, the upgrade can be initiated by skipping the preflight check. ")
+		klog.V(1).Infoln("During the upgrade, kubeadm will migrate the configuration while leaving the listed plugin configs untouched, but cannot guarantee that they will work with the newer version of CoreDNS.")
```

Comment:
this seems like detailed information that should be visible outside of klog and verbosity levels.
i think it makes sense to print it using `fmt.Print*()`. we have similar for `kubeadm init`.

i'd also print it like so trying to keep the lines in the source code shorter.

```
fmt.Println("The corefile contains plugins that kubeadm/CoreDNS does not know how to migrate. " +
	"Each plugin listed should be manually verified for compatibility with the newer version of CoreDNS. " +
	"...")
```

separate `fmt.Println()` seem appropriate if a sentence should be a separate paragraph.